src/EPR.Calculator.API.UnitTests/Builder/Summary/CalcResultSummaryBuilderTests.cs

```diff
@@ -193,7 +193,7 @@ public void TestInitialize()
                             NorthernIrelandTotal=0.15M,
                             ScotlandTotal=0.15M,
                             WalesTotal=020M,
-                            Name="Test",
+                            Name="1 + 4 Apportionment %s",
```

Comment:
Please use constant, We already have a constant for this

Reply:
implement that Uday